### DISCUSSION TITLE
Released version please provide a detailed file name.

### DISCUSSION THREAD

**Discussion by fenginsc** [vlang/v#14634] [Ideas]
Every time I download from the release page, it is the files v_linux.zip, v_macos.zip, v_windows.zip, and then I have to open it every time to know what version it is.
If possible, please include the file version number or release timestamp when publishing, thank you.
for example:
v_linux_0.2.2.zip, v_macos_0.2.2.zip, v_windows_0.2.2.zip
v_linux_weekly.2022.07.zip, v_macos_weekly.2022.07.zip, v_windows_weekly.2022.07.zip

**Comment by medvednikov** [comment]
I agree. Will do :)
